### PR TITLE
feat: frontend support for login rule audit events

### PR DESCRIPTION
**Pull request by nklaassen**
This PR adds frontend support to nicely display login rule audit events. I followed the guide [here](https://github.com/gravitational/teleport/tree/master/web#adding-an-audit-event) and mostly copied the types used for the role create/delete events.

~So far I've skipped steps 5 and 6 in the guide which add example events to the storybook because the storybook is currently broken since the webapps merge, I can add those once it is fixed.~ Example events have now been added to the test fixtures. `yarn test && yarn lint` passes locally.

Screenshot 2023-01-25 at 11 22 25 AM

### CODE REVIEW

**Comment by nklaassen** [comment]
> Adding the example events to https://github.com/gravitational/webapps/blob/master/packages/teleport/src/Audit/fixtures/index.ts also includes them in the tests, so it might be worth adding them for now

The tests fail without `yarn test-update-snapshot` which currently doesn't exist in the project root and I'm not sure I can run it in the web/ subdir? Sounds like @hatched is posting a storybook fix pretty soon so I'll hold off merging this until I can add that.

**Review by zmb3 — APPROVED** [review]
LGTM pending the storybook part

**Comment by ryanclark** [comment]
> The tests fail without `yarn test-update-snapshot` which currently doesn't exist in the project root and I'm not sure I can run it in the web/ subdir? Sounds like @hatched is posting a storybook fix pretty soon so I'll hold off merging this until I can add that.

`yarn test -u` from the root works

File: web/packages/teleport/src/Audit/fixtures/index.ts
Comment (ryanclark) [inline web/packages/teleport/src/Audit/fixtures/index.ts:2687]:
```suggestion
    code: 'TLR01I',
```

Comment (nklaassen) [reply web/packages/teleport/src/Audit/fixtures/index.ts]: 🤦‍♂️ thanks, fixed

**Review by ryanclark — APPROVED** [review]

**Pull request merged**
